src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,15 @@
+package seedu.addressbook.data.person;
+
+/**
+ * A read-only immutable interface for objects which can be converted to a printable string representation
```

Comment:
:+1: 
it would be preferable for the comment to be in the active verb form. this however is a good precise comment!